backend/src/routes/api/cluster-settings/clusterSettingsUtils.ts

```diff
@@ -28,10 +32,30 @@ export const updateClusterSettings = async (
 ): Promise<{ success: boolean; error: string }> => {
   const coreV1Api = fastify.kube.coreV1Api;
   const namespace = fastify.kube.namespace;
-  const { pvcSize, cullerTimeout, userTrackingEnabled, notebookTolerationSettings } = request.body;
+  const {
+    pvcSize,
+    cullerTimeout,
+    userTrackingEnabled,
+    notebookTolerationSettings,
+    modelServingPlatformEnabled,
+  } = request.body;
   const dashConfig = getDashboardConfig();
   const isJupyterEnabled = checkJupyterEnabled();
   try {
+    if (
+      modelServingPlatformEnabled.kServe !== !dashConfig.spec.dashboardConfig.disableKServe ||
```

Comment:
it would be nice to have an utility method for this, since we are checking this same logic in several places (i.e. Custom Serving Runtimes, then later for platform selection in projects, here...)

Reply:
This is only for cluster settings at the backend, it won't be used anywhere else. All the other places will check the dashboard config feature flags directly. Only cluster settings will need to compare the settings with the dashboard config.

Reply:
mmm ok, that sounds fair